### ISSUE TITLE
Weird behaviour with stack-chain

### ISSUE THREAD

**Issue by julien-f** [mafintosh/why-is-node-running#3]
```js
require('why-is-node-running')
require('stack-chain')

console.log(process) // Accessing process!
```

```
> node index.js
/tmp/node_modules/stackback/index.js:33
    err.stack;
       ^

TypeError: Cannot redefine property: _sb_callsites
```

**Comment by AndreasMadsen** [comment]
I think this more of an issue between stack-chain and stackback. Looking at stackback I think it is mostly stack-chain there is at fault (well, that is always my go to assumtion, so it doesn't say much).

**Comment by AndreasMadsen** [comment]
Published a new version of `stack-chain` (1.3.6) that can work around this type of conflicts. It should work now.

**Comment by julien-f** [comment]
Thanks!
